lab/requirements.md

```diff
@@ -0,0 +1,51 @@
+# OPI Lab Requirements
+
+## Access
+
+- SSH connection is available through VPN.
+- Emergency console can be enabled for remote access in case of issue.
+
+## Backup
+
+- All critical servers must be backed up.
+- Restoration of critical servers must be done within a couple of hours.
+- Hosting provider must be trained to do backup/restore.
+
+## Capacity
+
+- Wait queue for automatic tests should be less than 1 hour.
```

Comment:
This depends on the amount of HW we have, the test run time, and the number of patches submitted. Hard to put something here IMHO as those variables might be out of our control.

Reply:
To a first order, number of test configs needed = average arrival rate of requests (GitHub PR rate) x average time to service each request (build+test time).   Request arrival rate can be derived by GitHub PR timestamps.  Validation folks should know build + test times.  So the data is there.

Not looking for any detailed analysis here.  Just use above formula as a ball park estimate.  Then iterate based on actual measurements of the queuing delays to adjust number of test configs as needed.  I would say if folks need to wait more than one hour to have their CI results, lab capacity needs are woefully underestimated.

Reply:
What about "Wait queue for automatic tests should be targeted to be less than 1 hour." ?

Reply:
@mestery Am assuming you are OK with the "< 1 hour" requirement, yes?   I would say it would be better to be < 15 minutes.  Folks don't really want automated test results for their PRs to take more than 15 minutes, right?